tests/helpers/drupal-commands.js

```diff
@@ -19,6 +19,18 @@ module.exports = {
     return await page.goto(`${lang_prefix}/node/${nid}/edit?destination=admin/content`);
   },
 
+  /**
+   * Logs out the current user.
+   * @param page Page object.
+   * @param base_url Base URL.
+   * @return {Response} The response.
+   */
+  userLogout: async (page, base_url) => {
+    await page.goto(`${base_url}/user/logout`);
+    // Logout has a confirmation page since Drupal 10.3.
+    await page.locator('input[value="Log out"]').click();
+  },
+
```

Comment:
This is not a (drush) command, so it does not belong in drupal-commands.js.
It should be in test-methods.js, probably close to iLoginAs() -- and probably named iLogOut. (I don't necessarily agree with the naming, but at least it's consistent?(

Reply:
Right... I was "blind". Moved it to the correct place and renamed.